src/Extension/CoreExtension.php

```diff
@@ -1666,18 +1666,10 @@ function twig_check_arrow_in_sandbox(Environment $env, $arrow, $thing, $type)
      */
     function hasAttribute($object, $attribute)
     {
-        if (!is_object($object)) {
-            return false;
-        }
-
-        try {
-            $object->$attribute;
-
-            return true;
-        } catch (Exception $exception) {
-            // do nothing
+        if (method_exists($object, 'hasAttribute')) {
+            return $object->hasAttribute($attribute);
```

Comment:
This seems not something generic or universal ... or is this method part of some standard ?

Reply:
I think most active record patterns have this method. Yii and Laravel do at least couldnt find anything for others

https://github.com/illuminate/database/blob/master/Eloquent/Concerns/HasAttributes.php#L45

Reply:
That's not something we will add to Twig core.